tsl/src/compression/compression.c

```diff
@@ -1450,3 +1452,47 @@ compression_get_toast_storage(CompressionAlgorithms algorithm)
 		elog(ERROR, "invalid compression algorithm %d", algorithm);
 	return definitions[algorithm].compressed_data_storage;
 }
+
+/* Get relstats from compressed chunk and insert into relstats for the
+ * corresponding uncompressed parent.
```

Comment:
Does uncompressed parent mean the hypertable?

Reply:
uncompressed parent = chunk for which this is the associated compressed chunk.

Reply:
This is confusing terminology. If I don't mistake compressed chunk inherits from compressed hypertable and the parent usually means the object to inherit from.

Reply:
modified comment.